cclient/types/types.c

```diff
@@ -108,3 +112,86 @@ int_array* string_to_int_array(char* in) {
   }
   return in_arr;
 }
+
+retcode_t flex_hash_to_trytes(flex_hash_t* hash, char* trytes) {
+  size_t trits_len = 0;
+  if (trytes == NULL) {
+    return RC_CCLIENT_FLEX_TRITS;
```

Comment:
`RC_CCLIENT_FLEX_TRITS` does not indicate that something went wrong, use `*NULL_PTR`